src/server.c

```diff
@@ -307,6 +307,12 @@ int dictSdsKeyCaseCompare(dict *d, const void *key1, const void *key2) {
     return strcasecmp(key1, key2) == 0;
 }
 
+/* Case insensitive key comparison */
+int hashsetStringKeyCaseCompare(hashset *hs, const void *key1, const void *key2) {
```

Comment:
The function dictSdsKeyCaseCompare above has this comment which is now outdated:

```
/* A case insensitive version used for the command lookup table and other
 * places where case insensitive non binary-safe comparison is needed. */
```

It's not used for the command lookup table anymore.